Makefile

```diff
@@ -11,6 +11,23 @@ dev:
 	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-development)
 	$(eval RESOURCE_NAME_PREFIX=s165d01)
 	$(eval ENV_SHORT=dv)
+	$(eval ENV_TAG=dev)
+
+.PHONY: test
+test:
+	$(eval DEPLOY_ENV=test)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-test)
+	$(eval RESOURCE_NAME_PREFIX=s165t01)
+	$(eval ENV_SHORT=st)
+	$(eval ENV_TAG=test)
+
+.PHONY: preprod
+preprod:
+	$(eval DEPLOY_ENV=preprod)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-test)
```

Comment:
Are we sure we want to deploy pre-prod environment to test subscription?. I remember Theo wanted it in production sub

Reply:
`preprod` uses the Test subscription on Find: https://github.com/DFE-Digital/find-a-lost-trn/#links-and-application-names

I think that's actually preferable, because it means devs can ssh/rails console into it without PIM

Keeping preprod in Test is fine :+1: